changelog/update-wordpress-url-dependency

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Update @wordpress/url to v3.7.1
```

Comment:
Shouldn't we write `Update @wordpress/date to v3.5.0` here?

This way, we can get the changelog entry to be correct at least.

Reply:
No need to change, I misunderstood. I can see that there are two changelog entries here, which explains the issue.